tasks/CleanUp.tex

```diff
@@ -65,7 +65,7 @@ \subsection*{Referee instructions}
 \subsection*{OC instructions}
 2 hours before the test:
 \begin{itemize}
-	\item Announce the starting location of the robot.
+	\item Announce location of garbage collection zone.
```

Comment:
Announce **the** location of **the** garbage...
Ist nicht deutsch!!! Du musst Artikel verwenden :)

Reply:
Ah I dont't think they are really required. But maybe a bit nicer to read.